src/csharp/Microsoft.Spark/Interop/Ipc/JvmBridge.cs

```diff
@@ -45,6 +47,8 @@ internal JvmBridge(int portNumber)
 
             _portNumber = portNumber;
             _logger.LogInfo($"JvMBridge port is {portNumber}");
+
+            _jvmThreadPool = new JvmThreadPool(this, TimeSpan.FromMinutes(30));
```

Comment:
Can we make this configurable using env variable? Also, make the default value to be either 5 min or 10 minutes?

Reply:
Where's the best place to put the config variable?  `SparkEnvironment.cs`?

Reply:
You can try this file for now: https://github.com/dotnet/spark/blob/master/src/csharp/Microsoft.Spark/Services/ConfigurationService.cs

Reply:
Moved the config to that file.